src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -12,9 +12,10 @@ internal static partial class PathInternal
     {
         // There is only one invalid path character in Unix
         private const char InvalidPathChar = '\0';
-        internal static readonly char[] InvalidPathChars = { InvalidPathChar };
-
+        
         internal static readonly int MaxComponentLength = Interop.Sys.MaxName;
+        
+        internal static char[] GetInvalidPathChars() => new char[] { InvalidPathChar };
```

Comment:
@stephentoub This is now a method rather than a field, since the `InvalidPathChars` field has been moved to the PathInternal.cs file.